R/build_collection.R

```diff
@@ -1,6 +1,32 @@
 
 
+#' Build full metadata collection
+#'
+#' @param id unique identifier for the collection
+#' @param type always set to "Collection" for collection items
+#' @param stac_version version of STAC to implement
+#' @param license collection license
+#' @param links list of related objects and their relation relationships
+#' @param title short title for the collection
+#' @param assets dictionary of asset objects that can be downloaded
+#' @param extent spatial and temporal extents
+#' @param keywords list of keywords describing the collection
+#' @param providers all organizations capturing or processing the data or the hosting provider
+#' @param summaries a map of property summaries, either a set of values, a range of values or a JSON schema
+#' @param description detailed multi-line description to fully explain the collection
+#' @param item_assets an optional dictionary of asset objects associated with the collection that can be downloaded or streamed, each with a unique key
+#' @param table_columns column names, type, and description for data table
+#' @param extensions a list of extensions the collection implements
+#' @param publications list of relevant publication information for the collection
+#'
+#' @return
+#' @export
+#'
+#' @examples
 build_collection <- function(id,
+                             type = 'Collection',
+                             stac_version = '1.0.0',
+                             license = 'proprietary',
```

Comment:
This is looking good! 

I think we should just make our default license CC0 (<https://creativecommons.org/publicdomain/zero/1.0/legalcode>) but it's just a default so doesn't really matter.